package.json

```diff
@@ -36,7 +36,6 @@
     "@types/node": "^12.12.21",
     "@types/react": "^17.0.2",
     "@types/react-dom": "^17.0.1",
-    "@types/lodash": "^4.14.170",
```

Comment:
Shouldn't we keep this? The Form and /apply rely on this.

Reply:
This was a redundant entry! Lodash types is still in package.json